solutions/swb-reference/src/constants.ts

```diff
@@ -61,6 +63,14 @@ const SolutionId: string = 'SO0231'; //TODO: retrieve value dynamically
 const SolutionName: string = 'Service Workbench on AWS v2'; //TODO: retrieve value dynamically
 const SolutionVersion: string = '2.0.0'; //TODO: retrieve value dynamically
 const ApplicationType: string = 'AWS-Solutions'; //TODO: retrieve value dynamically
+const customUserAgentString: string = `AwsSolution/${SolutionId}/${SolutionVersion}`;
+
+const customUserAgent: UserAgent = [
+  [
+    customUserAgentString.slice(0, customUserAgentString.lastIndexOf('/')),
```

Comment:
i agree that the string in L66 is sufficient. Originally had it since UserAgent has type UserAgentPair[] ([doc here](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/types/_aws_sdk_types.UserAgentPair.html))

But passing string to the sdkOptions is enough. I can take it out